### PR TITLE
Added support for nested locale strings

### PR DESCRIPTION
**Pull request by emilysgoddard**

### CODE REVIEW

**Comment by emilysgoddard** [comment]
Added support for nested locale strings

File: src/locales.js [hunk 1]
```diff
@@ -148,10 +149,30 @@ function readLocaleFiles(baseLocalePath, baseName, defaultLocale) {
...
   });
-
+ 
   return langs;
 }
 
+var flattenObject = function(ob) {
+	var toReturn = {};
+
+	for (var i in ob) {
```
Comment (spacerockzero) [inline src/locales.js]: Looks like the indent length might be too long, but sometimes github makes things look weird. If using 2-space indentation, it should look more consistent.

Comment (spacerockzero) [inline src/locales.js]: Do we know why json.parse wasn't working for us before?

Comment (spacerockzero) [inline src/locales.js]: Nvm. I just read the code.

**Comment by gechols** [comment]
The test case looks like an accurate representation of what we want to do.

**Comment by spacerockzero** [comment]
We have now verified this works as expected in the reference app locally.

**Pull request merged**